PokeDemo/Presenter/PokemonListPresenter.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  PokemonListPresenter.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 15/07/22.
+//
+
+import Foundation
+import UIKit
+
+protocol PokemonListPresenterDelegate: AnyObject {
+    func show(_ pokemons: [Pokemon])
+    func showAlert(title: String, message: String)
+}
+
+typealias PokemonListViewPresenterDelegate = PokemonListPresenterDelegate & UIViewController
```

Comment:
It does not make any sense to introduce at this stage. Remove

Reply:
Covered in e5b038e391ff3c33386eaab59afd64089109162d